src/main/java/software/amazon/awssdk/crt/http/HttpConnection.java

```diff
@@ -117,17 +138,61 @@ private HttpConnection(URI uri, ClientBootstrap bootstrap, SocketOptions socketO
                 clientBootstrap.native_ptr(),
                 socketOptions.native_ptr(),
                 useTls ? tlsContext.native_ptr() : 0,
+                windowSize,
                 uri.getHost(),
                 port));
 
         return connectedFuture;
     }
 
+    /**
+     * Schedules an HttpRequest on the Native EventLoop for this HttpConnection.
+     *
+     * @param request The Request to make to the Server.
+     * @param streamHandler The Stream Handler to be called from the EventLoop
+     * @throws CrtRuntimeException
+     * @return The HttpStream that represents this Request/Response Pair. It can be closed at any time during the
+     *          request/response, but must be closed by the user thread making this request when it's done.
+     */
+    public HttpStream executeRequest(HttpRequest request, CrtHttpStreamHandler streamHandler) throws CrtRuntimeException {
```

Comment:
This function is MakeRequest/make_request in other SDKs, this should probably match

Reply:
Fixed